developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.rst

```diff
@@ -7,13 +7,50 @@ Upgrade to Nextcloud 24
 General
 -------
 
-tbd
+info.xml
+^^^^^^^^
+
+Make sure your ``appinfo/info.xml`` allows for Nextcloud 24.
+
+.. code-block:: xml
+
+	<dependencies>
+	  <nextcloud min-version="21" max-version="24" />
+	</dependencies>
 
 Front-end changes
 -----------------
 
 tbd
 
```

Comment:
```suggestion
```

Reply:
those are for the front-end section, I'd leave that for now